packages/engine/tests/units/multiple_groups/mod.rs

```diff
@@ -0,0 +1,18 @@
+use crate::experiment::run_test_suite;
+
+// TODO: We should expand this test suite with more simulations, and ideally not need to duplicate
+//  the experiment projects such as Sugarscape
```

Comment:
We could point to the sugarscape simulation in the `examples` folder and add a small 5-step experiment. This however requires, that the `age.rs` behavior is replaced with `age.js` you provided. I think this should be part of Replace [Rust behaviors with JavaScript in example sims](https://app.asana.com/0/1199548034582004/1201832146336433/f) (_internal_).

Reply:
I agree that's a possibility, but I think we should think about the indirection that adds and how it might be a bit unclear if everything else is co-located and then we have a few tests that point to another folder. 

One might try and update the examples test and be surprised when it affects the results of a unit test, etc.